packages/backend/src/tileDetails/models/errors.ts

```diff
@@ -0,0 +1,13 @@
+export class TileDetailsNotFoundError extends Error {
```

Comment:
Suggestion: 
I think you should implement HttpError object from `@map-colonies/error-express-handler` on this error. 
Then, add `public readonly status = httpStatus.NOT_FOUND;` - instead of catching the error and setting the returned status code manually.

Reply:
I disagree.
`TileDetailsNotFoundError` is being used in `TileDetailsManager` which shouldn't be decoupled to an http server. thus implementing errors with http properties would not be best practice.
for example, this becomes handy when we're unit testing this function and are aware of the exact error that was thrown in the relevant case.